lib/commands/general.js

```diff
@@ -1,7 +1,62 @@
 import _ from 'lodash';
+import { fs, tempDir } from 'appium-support';
+import log from '../logger';
 
 let commands = {}, helpers = {}, extensions = {};
 
+
+commands.getStrings = async function (language) {
```

Comment:
Is there no way to not copy and paste this from `appium-uiautomator2-driver`?

Reply:
I think this part of code might be changed in the future. also, it is not very clear where to put the common code

Reply:
I think it would be an overkill